spec/connect/client_spec.rb

```diff
@@ -716,6 +713,16 @@
 
       it { expect { subject }.to raise_error(::SUSE::Connect::SystemNotRegisteredError) }
     end
+
+
+    context 'when running on on-demand instance' do
+      before do
+        allow(File).to receive(:exist?).and_call_original
```

Comment:
Shouldn't this be treated as an `expect`?

Reply:
Depends on your view on whether expectations should be allowed in `before` block. Rubocop complains about that, though.